analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AssignmentInsideSubExpression.CSharp10.cs

```diff
@@ -29,4 +29,10 @@ private void Method(int arg = 42)
         var i = 0;
         Method(i = 42); // Noncompliant
     }
+
+    private void Method2()
+    {
+        int y, z;
+        var x = (y, z) = (16, 23);
```

Comment:
I think we should have `(y, var x) = (16, 23)` in order to use the latest language feature for this test case.